dataflow-guides/cloud-backup.md

```diff
@@ -1,45 +1,60 @@
-Description of guide
+This guide will walk you through how to configure and test running a dataflow with cloud recovery. To do so we will start with an AWS kubernetes cluster and then migrate the same dataflow to a Gooogle Cloud Kubernetes cluster. Having an account with a cluster in each is a prerequisite for trying this.
+
+Cloud recovery is a capability that allows the state persisted in the local worker to be written to object storage like s3. In the event of a node dying, the cluster going down or some other catastrophic failure, you will be able to recover dataflows from this bucket. This can also aid in Kubernetes cluster migration without the loss of state.
 
 ## Steps
 
 1. Deploy a dataflow with recovery and cloud backup enabled
+
 ```sh
 waxctl df deploy simple_slow.py \
   --platform \
   --name demo01 \
   --recovery=true \
   --recovery-backup=true \
-  --recovery-backup-s3-url=s3://litestream-tests/demo01 \
+  --recovery-backup-s3-url=s3://backup-tests/demo01 \
   --recovery-backup-s3-k8s-secret=aws-credentials
 ```
 
 2. Show the state
 
-https://dashboard.lab.bytewax.io/
+You can view the dataflow and the state object via the platform dashboard.
```

Comment:
In the example we can show that the state was recovered looking the logs. But it's because the dataflow is very simple and it only outputs a sequence.